rotkehlchen/globaldb/updates.py

```diff
@@ -66,8 +66,7 @@ def _force_remote(cursor: sqlite3.Cursor, local_asset: Asset, full_insert: str)
 
     May raise an sqlite3 error if something fails.
     """
-    cursor.executescript('BEGIN TRANSACTION; PRAGMA foreign_keys=off; COMMIT;')
-    # Delete the old entry
+    cursor.executescript('PRAGMA foreign_keys = OFF;')
```

Comment:
There is other places in the code where this is used. Please change it everywhere if it's used wrong in other places too

Reply:
In the other places where this is used the use is correct